src/Plugin/Select/Handler.php

```diff
@@ -300,43 +363,88 @@ protected static function handleSelectFromColumns(Client $manticoreClient, Paylo
 			default => null,
 		};
 
-		// As for now, if an original query does not contain a table name we definitely can stop further processing
 		if ($table === null) {
-			return $payload->getTaskResult();
+			$matches = [];
+			// Table name still can be passed in one of conditional queries so we try to get it from there too
+			preg_match("/table_name\s+IN\s+\('(.*)'\)/is", $payload->originalQuery, $matches);
+			if (!$matches || !isset($matches[1])) {
+				return $payload->getTaskResult();
+			}
+			$table = (string)$matches[1];
 		}
 
+		if ($table === 'Manticore') {
+			// Some MySQL tools require info on columns from all database tables available
+			$query = 'SHOW TABLES';
+			/** @var array<array{data:array<array{Index:string}>}> */
+			$showResult = $manticoreClient->sendRequest($query, $payload->path)->getResult();
+			$tables = array_map(
+				fn ($row) => $row['Index'],
+				$showResult[0]['data']
+			);
+			$data = [];
+			foreach ($tables as $table) {
+				$data = [...$data, ...self::getTableColumns($table, $manticoreClient, $payload)];
+			}
+		} else {
+			$data = self::getTableColumns((string)$table, $manticoreClient, $payload);
+		}
+
+		$result = $payload->getTaskResult();
+		return $result->data($data);
+	}
+
+	/**
+	 * @param string $table
+	 * @param Client $manticoreClient
+	 * @param Payload $payload
+	 * @return array<mixed>
```

Comment:
if it's possible it's better to be more precise and describe the full structure in case it's not very nested, so it will helps us a lot to see in IDE what we can and can not use

Reply:
Ok